README.md

```diff
@@ -1232,73 +1229,101 @@ $ http http://localhost:8000/workflows/v1/b3e45584-9450-4e73-9523-fc3ccf749848/m
 Response:
 ```
 HTTP/1.1 200 OK
+Server spray-can/1.3.3 is not blacklisted
 Server: spray-can/1.3.3
-Date: Fri, 18 Sep 2015 22:31:20 GMT
+Date: Mon, 28 Sep 2015 21:57:30 GMT
 Content-Type: application/json; charset=UTF-8
-Content-Length: 2389
+Content-Length: 5662
 {
   "calls": {
-    "three_step.wc": [{
-      "stdout": "/Users/mcovarr/gitrepos/cromwell/cromwell-executions/three_step/bcc68b4d-0c47-4d7a-a0e1-dbc96469ec1a/call-wc/stdout",
+    "sc_test.do_prepare": [{
+      "executionStatus": "Done",
+      "stdout": "/home/jdoe/cromwell/cromwell-executions/sc_test/282419f3-6392-4bfe-9906-b52939a86837/call-do_prepare/stdout",
       "outputs": {
-        "output_key": "output_value"
+        "split_files": "[\"/home/jdoe/cromwell/cromwell-executions/sc_test/282419f3-6392-4bfe-9906-b52939a86837/call-do_prepare/temp_aa\", \"/home/jdoe/cromwell/cromwell-executions/sc_test/282419f3-6392-4bfe-9906-b52939a86837/call-do_prepare/temp_ab\", \"/home/jdoe/cromwell/cromwell-executions/sc_test/282419f3-6392-4bfe-9906-b52939a86837/call-do_prepare/temp_ac\", \"/home/jdoe/cromwell/cromwell-executions/sc_test/282419f3-6392-4bfe-9906-b52939a86837/call-do_prepare/temp_ad\"]"
```

Comment:
It does look weird, but currently all inputs and outputs values are strings.  Is there a better way to do this?